docs/overrides.md

```diff
@@ -319,18 +321,22 @@ It's also possible to specify a namespace for a packaged Helm chart to be instal
 ```yaml
 kind: UDSBundle
 metadata:
-   name: example-bundle
-   version: 0.0.1
+  name: example-bundle
+  version: 0.0.1
 
 packages:
-   - name: helm-overrides-package
-     path: "../../packages/helm"
-     ref: 0.0.1
-     overrides:
-        podinfo-component:
-          unicorn-podinfo:
-             namespace: custom-podinfo # custom namespace!
-             values:
-                - path: "podinfo.replicaCount"
-                  value: 1
+  - name: helm-overrides-package
+    path: "../../packages/helm"
+    ref: 0.0.1
+    overrides:
+      podinfo-component:
+        unicorn-podinfo:
+          namespace: custom-podinfo # custom namespace!
+          values:
+            - path: "podinfo.replicaCount"
+              value: 1
 ```
+
+### View All Variables
+
+When working with a local or remote bundle you can view all overrides and zarf variables by running `uds inspect --list-variables <tarball>:ref`
```

Comment:
Slight correction, when running uds inspect --list-variables on a local tarball, no need to include the ref

Reply:
I think you can do something like `uds inspect --list-variables BUNDLE_TARBALL|OCI_REF]`